application/language/spanish/statistics_lang.php

```diff
@@ -2,43 +2,43 @@
 
 defined('BASEPATH') OR exit('No direct script access allowed');
 
-$lang['statistics_statistics'] = 'Statistics';
+$lang['statistics_statistics'] = 'Estadísticas';
 
-$lang['statistics_explore_the_logbook'] = 'Explore the logbook.';
+$lang['statistics_explore_the_logbook'] = 'Explore el libro de guardia.';
 
-$lang['statistics_years'] = 'Years';
-$lang['statistics_modes'] = 'Mode';
-$lang['statistics_bands'] = 'Bands';
+$lang['statistics_years'] = 'Años';
+$lang['statistics_mode'] = 'Modo';
+$lang['statistics_bands'] = 'Bandas';
 $lang['statistics_qsos'] = 'QSOs';
-$lang['statistics_unique_callsigns'] = 'Unique callsigns';
+$lang['statistics_unique_callsigns'] = 'Indicativos únicos';
 
 $lang['statistics_total'] = 'Total';
 
-$lang['statistics_year'] = 'Year';
+$lang['statistics_year'] = 'Año';
 
-$lang['statistics_number_of_qso_worked_each_year'] = "Number of QSOs worked each year";
-$lang['statistics_number_of_qso_worked'] = "# of QSO's worked";
+$lang['statistics_number_of_qso_worked_each_year'] = "Número de QSOs logradas cada año";
+$lang['statistics_number_of_qso_worked'] = "# de QSOs logradas";
 
 /*
 *
 * Distances
 *
 */
 
-$lang['statistics_distances_worked'] = "Distances Worked";
-$lang['statistics_distances_part1_contacts_were_plotted_furthest'] = "contacts were plotted.<br /> Your furthest contact was with";
-$lang['statistics_distances_part2_contacts_were_plotted_furthest'] = "in gridsquare";
-$lang['statistics_distances_part3_contacts_were_plotted_furthest'] = "the distance was";
-$lang['statistics_distances_number_of_qsos'] = "Number of QSOs";
-$lang['statistics_distances_callsigns_worked'] = "Callsign(s) worked (max 5 shown)";
-$lang['statistics_distances_qsos_with'] = "QSOs with";
-$lang['statistics_distances_and_band'] = "and band";
+$lang['statistics_distances_worked'] = "Distancias Logradas";
+$lang['statistics_distances_part1_contacts_were_plotted_furthest'] = "contactos fueron dibujados.<br /> Su contacto más lejano fue con";
+$lang['statistics_distances_part2_contacts_were_plotted_furthest'] = "en gridsquare";
+$lang['statistics_distances_part3_contacts_were_plotted_furthest'] = "la distancia fue";
+$lang['statistics_distances_number_of_qsos'] = "Número de QSOs";
+$lang['statistics_distances_callsigns_worked'] = "Indicativo(s) logrados (máximo se muestran 5)";
```

Comment:
I would say just "máximo 5" or "se muestran máximo 5".

Also "trabajados" instead of "logrados".